pkg/runtime/component/wasm.go

```diff
@@ -0,0 +1,22 @@
+// This needs to be moved to contrib repo if accepted.
+// Put it here just for disscuss how to pass configuration to components.
```

Comment:
I would like that we have options in metadata then we can control it finer.

Reply:
Thanks for review. Both ways is ok with me, Changed to pass option via metadata.